src/EmployeeManagement/EmployeeManagement.IntegrationTest/Constants/TestConstants.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace EmployeeManagement.IntegrationTest.Constants
+{
+    public static class TestConstants
```

Comment:
Additional constants file is not needed move this to main project consts file, same will be needed for program.cs file later on